scripts/check-data-freshness.ts

```diff
@@ -86,3 +123,18 @@ test('All URLs linking to current programs have an OK response code', async tap
     }
   }
 });
+
+test('PDF returned readable text, able to be parsed', async tap => {
+  // Check that PDF link can be read and returned.
+  // This PDF may oneday no longer exist, and if this test fails, the url should be checked first.
```

Comment:
typo: one day

I feel a little weird about having a test that relies on a single PDF being available. Is this because you want there to be something exercising the code but we can't rely on the test working for all PDFs because of the timestamp issue?

Reply:
I actually found that the timestamp issue isn't a problem for this! It was surprising when I read over the text returned as actually just the text available in the PDF. The pdf-parse library was working successfully to scrape out the text in the file - what didn't work was the axios request for the PDF (with the wrong content header type and array buffer response not being properly defined). Post-that, I've found the PDF parsing library to work well at only returning the text (albeit, this doesn't help table-scraping the PDF files, since the text returned is pretty unformatted, but it's  a great improvement to before). 


For the test itself - I don't have a great idea of how to test this other than using a real web PDF, but from the overall suggestions you've provided below, I think that I can just ensure for PDF content, real content is returned, so this will become a non-issue.

Reply:
We should at least use a PDF hosted on one of our domains, if we're going to have a live-network-fetch automated test.